### PR TITLE
Fix #56: Add support for Artist.sortname field.

### LINKED ISSUE #56: Add "sortname" fields
See mopidy/mopidy#940.

Even if this doesn't get added to models, the fields can still be extracted from gstreamer tags passed to `Library.add()` in Mopidy v0.20. `Library.browse()` can then return results using
```
ORDER BY coalesce(sortname, name)
```

### PR DESCRIPTION
**Pull request by tkem**
Finally making use of `models.Artist.sortname`. Not sure if this should be enabled by default, though, since browsing artists looks kind of weird if only some of your music files contain "sortname" tags.

### CODE REVIEW

File: mopidy_local_sqlite/schema.py [hunk 1]
```diff
@@ -169,8 +169,7 @@ def load(c):
     user_version = c.execute('PRAGMA user_version').fetchone()[0]
     if not user_version:
         logger.info('Creating SQLite database schema v%s', schema_version)
-        script = os.path.join(sql_dir, 'create-v%s.sql' % schema_version)
-        c.executescript(open(script).read())
+        c.executescript(open(os.path.join(sql_dir, 'schema.sql')).read())
```
Comment (jodal) [inline mopidy_local_sqlite/schema.py]: Not that we'll run on PyPy in the immediate future, but PyPy would issue a warning about a leaking resource here. Alternative version:

``` python
with open(os.path.join(sql_dir, 'schema.sql')) as fh:
    c.executescript(fh.read())
```

**Comment by jodal** [comment]
LGTM :-)

File: mopidy_local_sqlite/schema.py (hunk 1 above)
Comment (tkem) [inline mopidy_local_sqlite/schema.py]: PyPy or not, I like your version a lot better; should have thought of that myself ;-)

**Comment by tkem** [comment]
Took the opportunity to clean up a little more:
- more DRY
- sanity check so buggy upgrade scripts don't lead to an endless loop (easily)

**Pull request merged**